### ISSUE TITLE
About train parameter

### ISSUE THREAD

**Issue by jacob-kang** [bethgelab/robust-detection-benchmark#7]
Hello! Thanks for your great paper.
I'm researching about shape vs texture.
I just made Stylized-COCO dataset.

In your paper, you train a model to stylized-COCO 2017 and validate to COCO 2017 (No stylized).
And you've shared your pretrained model in your texture vs shape repo.
So I used your SIN pretrained weight as pretrained model to train Stylized-COCO.

But the problem is, It is too low the accuracy to validation set.

In my thoughts, I use SIN pretrained weight, So it should be high accuracy or at least same with your paper score.
Do you know why this is occured? or Should I train the model even I use SIN pretraine model?

Thanks.

**Comment by rgeirhos** [comment]
See Table 2 of https://openreview.net/pdf?id=Bygh9j09KX: the best MS COCO performance is achieved when using a backbone network trained jointly on SIN+IN then finetuned on IN (model_C in https://github.com/rgeirhos/texture-vs-shape/blob/master/models/load_pretrained_models.py).
